docs/tooling/cli-guides/create-a-validator.md

```diff
@@ -0,0 +1,102 @@
+# Start a Validator with 1-command
+
+This page will guide you to start your own Avalanche Validator through Avalanche-CLI with only
+1 command!
+
+Currently, we are only supporting Fuji network with Mainnet support coming in the near future!
+
+:::warning
+
+ALPHA WARNING: This command is currently in experimental mode. Proceed at your own risk.
+
+:::
+
+## Prerequisites
+
+Before we begin, you will need to:
+
+- Create an AWS account and have an AWS `credentials` file in home directory, More info can be
+  found [here](https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html#file-format-creds)
+- Install [Terraform](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli)
+- Install [Ansible](https://adamtheautomator.com/install-ansible/)
+
+Currently, we are only supporting AWS cloud service, but we plan to add more cloud services in the
+near future.
+
+## Starting a Validator
+
+To start an Avalanche validator, run:
```

Comment:
Choose a name to refer to the set of validators you will manage. For this example we will use `myValidatorsCluster`.

Reply:
i agree with raymond